public/appConfig.js

```diff
@@ -0,0 +1,9 @@
+var appConfig = {
```

Comment:
Same as this one - we usually have an `appConfig.example.js` as an example but have the `appConfig.js` ignored since it's account specific usually.

Reply:
Solved in c867f67